smartsim/_core/_cli/clean.py

```diff
@@ -53,7 +53,7 @@ def clean(self, _all=False):
 
         bin_path = self._core_path / "bin"
         if bin_path.is_dir() and _all:
-            files_to_remove = ["redis-server", "redis-cli"]
+            files_to_remove = ["redis-server", "redis-cli", "keydb-server", "keydb-cli"]
```

Comment:
Probably out of scope for this PR, but something to keep in mind. We could add an artifact manifest to each of the builder objects. This would help us both sanity check that the build completely successfully (i.e. check every artifact and make sure it exists) and also help us simplify the clean stage by simply looping through all the builder objects and removing the artifacts.

Maybe write that up as a ticket?

Reply:
@Spartee :  Sure, I'll use `redis_builder` as an example:
```
class redis_builder():
  _expected_executable_targets = ['redis-cli', redis-server']
  def check_install(self):
    targets_installed = True
    for target in self._expected_targets:
      targets_installed = targets_installed and shutil.which(target)
      logger.warning(f'{target} could not be found')
  def clean(self):
    for target in _expected_executable_targets:
       shutil.rm(target)
```

Reply:
I like it. Thanks for clarifying